content/dev-prod-parity.md

```diff
@@ -1,76 +1,73 @@
 ## X. Dev/prod parity
+
 ### Keep development, staging, and production as similar as possible
 
-Historically, there have been substantial gaps between development (a developer making live edits to a local [deploy](./codebase.md) of the app) and production (a running deploy of the app accessed by end users).  These gaps manifest in three areas:
+#### 1. A twelve-factor app minimizes the gaps between development, staging, and production.
+
+Historically, there have been substantial gaps between development (a developer
+making live edits to a local [deploy](./codebase.md) of the app) and production
+(a running deploy of the app accessed by end users). These gaps manifest in
+three areas:
+
+- **The time gap**: A developer may work on code that takes days, weeks, or even
+  months to go into production.
+- **The personnel gap**: Developers write code, ops engineers deploy it.
+- **The tools gap**: Developers may be using a stack like Nginx, SQLite, and
+  OS X, while the production deploy uses Apache, MySQL, and Linux.
 
-* **The time gap**: A developer may work on code that takes days, weeks, or even months to go into production.
-* **The personnel gap**: Developers write code, ops engineers deploy it.
-* **The tools gap**: Developers may be using a stack like Nginx, SQLite, and OS X, while the production deploy uses Apache, MySQL, and Linux.
+**The twelve-factor app is designed for
+[continuous deployment](http://avc.com/2011/02/continuous-deployment/) by
+keeping the gap between development and production small.**
 
-**The twelve-factor app is designed for [continuous deployment](http://avc.com/2011/02/continuous-deployment/) by keeping the gap between development and production small.**  Looking at the three gaps described above:
+|                               | Traditional app  | Twelve-factor app      |
+| ----------------------------- | ---------------- | ---------------------- |
+| **Time between deploys**      | Weeks            | Hours                  |
+| **Code authors vs deployers** | Different people | Same people            |
+| **Dev vs production**         | Divergent        | As similar as possible |
 
-* Make the time gap small: a developer may write code and have it deployed hours or even just minutes later.
-* Make the personnel gap small: developers who wrote code are closely involved in deploying it and watching its behavior in production.
-* Make the tools gap small: keep development and production as similar as possible.
+##### Guidance
 
-Summarizing the above into a table:
+- Make the time gap small: a developer may write code and have it deployed hours
+  or even just minutes later.
+- Make the personnel gap small: developers who wrote code are closely involved
+  in deploying it and watching its behavior in production.
```

Comment:
I think this is a good practice, but not sure how people organize part of a 12 factor app.  For example pipelines that do package multi changes into releases vs single release per update